### ISSUE TITLE
restrict counts in callback of waveforms

### ISSUE THREAD

**Issue by aschaelicke** [pyepics/pyepics#72]
Hallo,

we are using pyepics for monitoring large waveforms (25Msamples, 16bit, @ 1Hz or 20Msampels, 16bit @ 2Hz). In general we get good performance by activate auto monitoring and adding a callback to receive the data.
But in order to reduce the network load it was beneficial to tell the subscription how many samples are actually required.

In fact the *calib* already supports this via the *count* argument
```python
    ret = libca.ca_create_subscription(ftype, count, chid, mask,
                                       _CB_EVENT, uarg, ctypes.byref(evid))
```
So we modified  *ca.py* and *pv.py* to support this argument.

Would it be possible to add this to an official release?
[patch_pyepics3.2.6_count.txt](https://github.com/pyepics/pyepics/files/583273/patch_pyepics3.2.6_count.txt)

**Comment by newville** [comment]
@aschaelicke this looks pretty straightforward.   I might nitpick that the default value for count better be "None" to signal "invalid" than -1.

Are you willing / able to make an actual Pull Request?   If not, I can apply these changes.

**Comment by aschaelicke** [comment]
done in pull request #73

**Comment by newville** [comment]
now fixed with #73
